src/internal/deploy/assets/assets.go

```diff
@@ -730,6 +743,58 @@ func PachdDeployment(opts *AssetOpts, objectStoreBackend Backend, hostPath strin
 	}
 }
 
+// EnterpriseService returns a service for the Enterprise Server
+func EnterpriseService(opts *AssetOpts) *v1.Service {
+	prometheusAnnotations := map[string]string{
+		"prometheus.io/scrape": "true",
+		"prometheus.io/port":   strconv.Itoa(PrometheusPort),
+	}
+	return &v1.Service{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Service",
+			APIVersion: "v1",
+		},
+		ObjectMeta: objectMeta(enterpriseServerName, labels(enterpriseServerName), prometheusAnnotations, opts.Namespace),
+		Spec: v1.ServiceSpec{
+			Type: v1.ServiceTypeNodePort,
+			Selector: map[string]string{
+				"app": enterpriseServerName,
+			},
+			Ports: []v1.ServicePort{
+				// NOTE: do not put any new ports before `api-grpc-port`, as
+				// it'll change k8s SERVICE_PORT env var values
+				{
+					Port:     650, // also set in cmd/pachd/main.go
+					Name:     "api-grpc-port",
+					NodePort: 31650,
+				},
+				{
+					Port:     651, // also set in cmd/pachd/main.go
+					Name:     "trace-port",
+					NodePort: 31651,
+				},
+				{
+					Port:     auth.OidcPort,
```

Comment:
So this runs it's own Dex.  I assume the configuration for that will be the same as for pachd?

Reply:
I'm not sure what you mean.